frontend/chat-plugin/src/airyRenderProps/AiryHeaderBar/index.tsx

```diff
@@ -1,28 +1,30 @@
 import React from 'react';
 import style from './index.module.scss';
+import {ReactComponent as CloseButton} from '../../../../assets/images/icons/close-button.svg';
+import {ReactComponent as MinimizeButton} from '../../../../assets/images/icons/minimize-button.svg';
 
-type Props = {
+type AiryHeaderBarProps = {
   toggleHideChat: () => void;
+  setShowModal: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
-const AiryHeaderBar = (props: Props) => {
+const AiryHeaderBar = (props: AiryHeaderBarProps) => {
   return (
-    <div className={style.header}>
-      <div className={style.headerInfo}>
-        <h1 className={style.headerTitle}>Customer Chat</h1>
+    <>
+      <div className={style.header}>
+        <div className={style.headerInfo}>
+          <h1 className={style.headerTitle}>Customer Chat</h1>
+        </div>
+
+        <button className={style.minimizeButton} onClick={() => props.toggleHideChat()} title="Minimize chat">
```

Comment:
You can also use`onClick={props.toggleHideChat}` It avoids unnecessary re-renders and reads better